nufi-official/flow-ispo
# Prototype of withdrawing user's flow from ISPO they joined

How to test:

Needs to be tested on flow testnet as emulator doesn't support staking

```
yarn install
cp .env.example .env // sets up config for testnet
yarn dev:testnet // hosts the app at http://localhost:3000
```
Use a clean testnet flow account, create an ISPO, join it and then try to withdraw flow from it in `My participations` screen - it will go through only the first time, as other times it will clash, see https://github.com/onflow/flow-core-contracts/blob/49b5accb2a84a6a2b113c858eca638a9a2a6aebb/contracts/FlowStakingCollection.cdc#L318
